samples/peripheral/802154_phy_test/sample.yaml

```diff
@@ -9,4 +9,20 @@ tests:
       - nrf21540dk_nrf52840
       - nrf5340dk_nrf5340_cpunet
     platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpunet
-    tags: ci_build
+    tags: ci_rs_build ci_rs_weekly
```

Comment:
Have you removed the `ci_build` tag intentionally?

Reply:
ci_build please change it to `ci_build ci_rs_weekly`

Reply:
I updated now, thanks!